app/services/AtsListService.scala

```diff
@@ -163,9 +165,13 @@ class AtsListService @Inject()(
       case (Left(_), identifier) =>
         auditService.sendEvent(
           AuditTypes.Tx_FAILED,
```

Comment:
Its not a failure if this user does not have a UTR in this scenario, you should leave it as a `Tx_SUCCEEDED`